.github/workflows/checks-all.yml

```diff
@@ -262,6 +262,49 @@ jobs:
           rust_backtrace=1 cargo test --test eth_movement -- --nocapture --test-threads=1 
         "
 
+  bridge-movement-eth:
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-22.04
+            arch: x86_64
+            runs-on: buildjet-8vcpu-ubuntu-2204
+
+    runs-on: ${{ matrix.runs-on }}
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Install Movement CLI
+      run: |
+        sudo apt-get update 
+        sudo apt-get install -y build-essential
+        sudo apt-get install -y binutils
+        sudo apt-get install -y lld
+        sudo apt-get install -y libudev-dev
+        sudo apt-get install -y libdw-dev
+        which ld
+        which lld
+        which gcc
+        which cc
+        echo $PATH
+        export GIT_CLONE_PROTECTION_ACTIVE=false
+        git clone https://github.com/movementlabsxyz/aptos-core/ 
+        cd aptos-core
+        cargo build -p movement
+        sudo cp target/debug/movement /usr/local/bin/
+        cd -
+
+    - name: Install Nix
+      uses: DeterminateSystems/nix-installer-action@main
+
+    - name: Run eth_movement tests
+      run: |
+        nix develop --command bash -c "
+          rust_backtrace=1 cargo test --test movement_eth -- --nocapture --test-threads=1 
+        "
```

Comment:
Hmm rather than a second job you could just append to the `nix develop --command` for `eth_movement` job and rename it `bridge client integration`, then we don't have to install the CLI twice, which is quite a resource heavy op.

Reply:
As in just a `&&` in the command so it will run both integration files first for eth_movement then movement_eth.

Reply:
Sure just did it as 

```
      run: |
        nix develop --command bash -c "
          rust_backtrace=1 cargo test --test eth_movement -- --nocapture --test-threads=1 
          && rust_backtrace=1 cargo test --test movement_eth -- --nocapture --test-threads=1
        "
```

Reply:
Needed the line breaks i think... should pass this time 

```
      run: |
        nix develop --command bash -c " \
          rust_backtrace=1 cargo test --test eth_movement -- --nocapture --test-threads=1 && \
          rust_backtrace=1 cargo test --test movement_eth -- --nocapture --test-threads=1 \
        "
```

Reply:
CI passes, good for re-review @0xmovses